src/app/utils/elementFactory.js

```diff
@@ -0,0 +1,23 @@
+const elementFactory = (tag, attributes, ...children) => {
```

Comment:
Some validation would be appreciated, consider it as additional function
https://stackoverflow.com/questions/33584700/how-to-check-if-dom-element-and-or-attribute-is-valid 👀

Reply:
Thanks, nice suggestion, will be considered in the future